cypress/integration/instruments.ts

```diff
@@ -161,15 +163,34 @@ context('Instrument tests', () => {
 
     cy.createCall(call2);
 
+    cy.navigateToTemplatesSubmenu('Proposal templates');
+
+    cy.contains('default template')
+      .parent()
+      .get("[title='Edit']")
+      .first()
+      .click();
+
+    cy.createTopic('Topic for questions');
+
+    cy.get('[data-cy=show-more-button]').last().click();
```

Comment:
It would be much better if we can target this item much better instead of using `.last()` and the same goes for the one below